src/services/formatting/formatting.ts

```diff
@@ -1117,6 +1117,25 @@ namespace ts.formatting {
         }
     }
 
+    export function getRangeOfEnclosingComment(sourceFile: SourceFile, position: number, onlyMultiLine: boolean): CommentRange | undefined {

```

Comment:
Why do we need this function? There's already an `isInComment` function exposed, and you don't use the range returned but just test if one was returned. Seems all we should care about is if the cursor is in a comment, right?